tests.py

```diff
@@ -4,8 +4,11 @@
 
 
 class TestMethods(unittest.TestCase):
-    def test_add(self):
+    def test_smile(self):
         self.assertEqual(awesome.smile(), ":)")
+        
+    def test_frown(self):
```

Comment:
```suggestion
    def test_frowny(self):
```

Call it this instead